### PR TITLE
Cu 861mhbt68 add storekeeper seo support no sitemap

### PR DESCRIPTION
**Pull request by ikulis**

### CODE REVIEW

File: src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/StoreKeeperSeoPages.php [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                $seo[StoreKeeperSeo::SEO_DESCRIPTION],
+                $attributes
+        );
+    }
+
+    public function saveCategorySeo($term_id)
+    {
+        $term = get_term($term_id);
+        $storekeeper_id = get_term_meta($term->term_id, 'storekeeper_id', true);
+        if (empty($storekeeper_id)) {
```
Comment (jnmrclmbsjse) [inline src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/StoreKeeperSeoPages.php:79]: Just a question, why do we only save it when no `storekeeper_id`?

**Review by jnmrclmbsjse — CHANGES_REQUESTED** [review]
Few questions and suggestions

File: src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/StoreKeeperSeoHandler.php [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+                $offer['availability'] = str_replace(
+                    'https://schema.org/',
+                    'http://schema.org/',
+                    $offer['availability']
+                );
+            }
+
+            // get minimum price instead of first variation
+            if ($product instanceof \WC_Product_Variable) {
+                $offer['price'] = $product->get_variation_price('min', false);
```
[the remark this answers is not in the document]
Comment (ikulis) [reply src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/StoreKeeperSeoHandler.php:114]: I know those are the defaults, but would rather be explicit (it case the default change in future)

File: src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/StoreKeeperSeoHandler.php [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+                                        $matched = true;
+                                    } else {
+                                        $markdown['gtin13'] = $barcode;
+                                        $matched = true;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
```
[the remark this answers is not in the document]
Comment (ikulis) [reply src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/StoreKeeperSeoHandler.php:204]: yep, it's you from 1,5years ago :D

File: src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/StoreKeeperSeoPages.php (hunk 1 above)
Comment (ikulis) [reply src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/StoreKeeperSeoPages.php:79]: if storekeeper_id is set than it should be managed in storekeeper, not in WooCommerce

**Review by jnmrclmbsjse — APPROVED** [review]

**Pull request merged**